app/screens/Charts/data.ts

```diff
@@ -0,0 +1,27 @@
+export const padding = {
+  left: 40,
+  bottom: 20,
+  right: 20,
+  top: 20,
+};
+export const xDomain = {
+  min: 2018,
+  max: 2020,
+};
+export const yDomain = {
+  min: 0,
+  max: 35,
+};
+export const scatter = {
+  width: 6,
+  height: 6,
+  rx: 2,
+};
+export const COLORS = {
+  CONCRETE: '#f3f3f3',
+  CONIFER: '#AADD33',
+};
+export const stroke = {
+  width: 5,
+};
+export const tickCount = 3;
```

Comment:
We need name it with UPPERCASE
padding -> PADDING
xDomain -> X_DOMAIN
yDomain -> Y_DOMAIN 
etc.